src/blocks/owc-openkaarten/streetmap/assets/scripts/vue/BaseSearchInput.vue

```diff
@@ -120,10 +125,17 @@ const searchStatus = computed(() => {
 .search-input {
   width: 100%;
   padding: 0.75rem 1rem;
-  padding-inline-end: 5rem; // Space for both buttons
+  padding-inline-end: 4.5rem; // Space for both buttons
   border: 1px solid #7a7a7a;
   border-radius: 4px;
   font-size: 1rem;
+  &::placeholder {
+    overflow: visible;
+    font-size: 14px;
+    @media only screen and (min-width: 768px) {
```

Comment:
Ik zou hier kiezen voor een @container query i.p.v. media-query. 

Maar is font-size: 1rem niet gelijk aan 16pixels?